environments/staging/common/rds.tf

```diff
@@ -0,0 +1,52 @@
+module "postgres" {
+  source = "../../common/rds"
+
+  environment    = var.environment
+  name           = "TradeTariffPostgres${title(var.environment)}"
+  engine         = "postgres"
+  engine_version = "13.11"
+
+  deletion_protection = false # while configuring
+
+  # smallest that supports encryption at rest and postgres 13.11
+  instance_type      = "db.t3.micro"
```

Comment:
Are we going to keep micro instances on staging and production too?
of course this is something we can tune based load/pef test outcomes in case we run.